ForumProject/projects/serializers.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import serializers
+from .models import Project
+from django.core.exceptions import ValidationError
+
+
+class ProjectSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Project
+        fields = [
+            'id',
+            'project_name',
+            'project_status',
+            'startup_id',
```

Comment:
If you followed the earlier advice and changed the `ForeignKey` field name from `startup_id` to `startup`, you need to update the serializer fields to reflect that change

Reply:
Done, thanks